fury/utils.py

```diff
@@ -715,6 +745,7 @@ def get_actor_from_primitive(vertices, triangles, colors=None,
 
     current_actor = get_actor_from_polydata(pd)
     current_actor.GetProperty().SetBackfaceCulling(backface_culling)
+    add_primitives_count_to_actor(current_actor, prim_count)
```

Comment:
You might see in the `utils.py` file that we have both the `get/set_polydata_property` (e.g., [get_polydata_normals](https://github.com/fury-gl/fury/blob/1d8abba390301ad3ffdd538af244afe05889b499/fury/utils.py#L406) and [set_polydata_normals](https://github.com/fury-gl/fury/blob/1d8abba390301ad3ffdd538af244afe05889b499/fury/utils.py#L543)) and the `property_from/to_actor` (e.g., [normals_from_actor](https://github.com/fury-gl/fury/blob/1d8abba390301ad3ffdd538af244afe05889b499/fury/utils.py#L1237) and [normals_to_actor](https://github.com/fury-gl/fury/blob/1d8abba390301ad3ffdd538af244afe05889b499/fury/utils.py#L1296)). The reason for this is that the *polydata* functions are meant to be used for tasks like actor creations since at some point you'll have access to the polydata before it goes to the mapper. While the *actor* functions are for specific use cases, they generally require getting the mapper to get the polydata to get the desired attribute finally. This process is inefficient. Therefore, I would recommend adding the respective `get/set_polydata_primitives_count` and using them here since you can access the polydata [here](https://github.com/fury-gl/fury/blob/1d8abba390301ad3ffdd538af244afe05889b499/fury/utils.py#L701).

Reply:
Hello @guaje, thanks for your review. I made all the changes you requested I believe. 
I also noticed this PR was not applicable on some actors while doing the tests. so I had them added.
I hope I didn't miss anything.

Reply:
Yeah, that has happened on several occasions. That was the main reason I asked you to test all the supported actors. In the past, we have assumed that some functions are supposed to work equally on all the actors, yet unfortunately, that's not the case since we have different methods to create them.